src/current/v23.2/configure-logs.md

```diff
@@ -483,6 +483,31 @@ sinks:
       channels: {INFO: DEV, ERROR: OPS}
 ~~~
 
+### Set timezone
+
+The log output formats [`crdb-v1`]({% link {{ page.version.version }}/log-formats.md %}#format-crdb-v1) and [`crdb-v2`]({% link {{ page.version.version }}/log-formats.md %}#format-crdb-v2) support the format option `timezone`. When specified, the corresponding timezone is used to produce the timestamp column. The value can be any [timezone name recognized by the Go standard library](https://github.com/arp242/tz/blob/bf333631bec4/list.go).
```

Comment:
This paragraph should link back to the 'Considerations' section on cockroach-debug-merge-logs.md because (If I'm understanding this correctly) once you set this setting it's a one-way ratchet, which will prevent you from ever doing `cockroach debug merge-logs` with a binary < v23.2.  Right?  Please correct me if I'm misunderstanding this!

Reply:
Discussed with @rmloveland. We decided that customers don't need to know the details of `merge-logs` which is used by TSE team.